### PR TITLE
Fix calico-node service failure on xenial due to ASCII locale

### PR DESCRIPTION
**Pull request by Cynerva**
Fixes https://bugs.launchpad.net/bugs/1911220 for Canal

### CODE REVIEW

File: wheelhouse.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 charms.templating.jinja2>=1.0.0,<2.0.0
-python-etcd>=0.4.0,<1.0.0
```
Comment (Cynerva) [inline wheelhouse.txt:2]: This was causing `charm build` to fail: https://paste.ubuntu.com/p/xxXnKx9mT5/

I'm pretty sure the charm doesn't even use this library, so it should be safe to remove.

Comment (johnsca) [reply wheelhouse.txt:2]: It looks like this [was added to be used indirectly by the flanneld service](https://github.com/charmed-kubernetes/layer-canal/commit/51cad6d38b4944790caa6b92cf7940f73641ee66#diff-6ce054f7bfdeb4690ef5fe38d2e88f00f9f3a91404cfa5a799cba2598f1d6ee2).  I'm not certain it's safe to remove.

Comment (johnsca) [reply wheelhouse.txt:2]: Also, same dependency [is used](https://github.com/charmed-kubernetes/charm-flannel/blob/master/wheelhouse.txt#L2) in the flannel charm.

Comment (Cynerva) [reply wheelhouse.txt:2]: In my test deployments on xenial and focal, flannel came up fine.

Regarding the linked commit, the commit message says:

> Removed "render script and run" bits, and favored python-etcd instead

If you look at reactive/flannel.py in the linked commit, you'll see that's where the library was used, via `import etcd`. As of today, that import is no longer present in the code, so it should be safe to remove.

How would the flanneld service even use this? The wheelhouse deps get installed in the charm's venv, but flanneld is not running in that context. Also, flanneld is a golang project, not python.

Comment (johnsca) [reply wheelhouse.txt:2]: As usual, your logic is flawless. :grin: :+1: from me.

**Review by johnsca — APPROVED** [review]

**Pull request merged**